temporal-docker-builds.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: temporal-docker-builds
+  # This project doesn't do releases and everything is commit based.
+  # This corresponds to commit 70f8b8bbda3723fe7c8f822cc8aa379598ac8c86
+  version: 0.0_git20231116
+  epoch: 0
+  description: Temporal service Docker images build
+  copyright:
+    - license: MIT License
+
+vars:
+  commit: 70f8b8bbda3723fe7c8f822cc8aa379598ac8c86
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 2eb296dc8493320261c762377356dedc23a4b3e4040502c21618e61b407a82a5
+      extract: "false"
+      uri: https://github.com/temporalio/docker-builds/archive/${{vars.commit}}.zip
+
+  - runs: |
+      unzip -q ${{vars.commit}}.zip
+      mv docker-builds-${{vars.commit}} docker-builds
+      rm *.zip
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/temporal-docker-builds
+
+      cp docker-builds/docker/entrypoint.sh "${{targets.destdir}}"/temporal-docker-builds
```

Comment:
```suggestion
      cp docker/entrypoint.sh "${{targets.destdir}}"/temporal-docker-builds
```

Reply:
similar to other copies as above

Reply:
Thanks @ajayk, just cleaning up all that!